File: src/Catalogo.Svc/Catalogo.Svc/Models/Mercadoria.cs
using System;
using Catalogo.Svc.Models.Base;

namespace Catalogo.Svc.Models
{
    public class Mercadoria : IEntity
    {
        public int ID { get; set; }
        public String Nome { get; set; }
        public double Preco { get; set; } 
    }
}

File: src/Catalogo.Mobile/app/src/main/java/br/com/grapp/catalogo/models/Mercadoria.java
package br.com.grapp.catalogo.models;

/**
 * Classe padrão JavaBean com seus atributos (privador) e métods gets e setters
 */
public class Mercadoria {
    private int _id;

    private String _nome;
    private double _preco;

    public Mercadoria(int ID, String nome, double preco) {
        _id = ID;
        _nome = nome;
        _preco = preco;
    }

    public int getID() {
        return _id;
    }

    public void setID(int ID) {
        _id = ID;
    }

    public String getNome() {
        return _nome;
    }

    public void setNome(String nome) {
        _nome = nome;
    }

    public double getPreco() {
        return _preco;
    }

    public void setPreco(double preco) {
        _preco = preco;
    }
}
